video.php

```diff
@@ -81,4 +81,4 @@ function youtube_video_valid($youtube_api_key,$video_id){
 		throw new Exception("Video is private $video_id");
```

Comment:
Not something that needs changing but we should get into the habit of adding a `\` to prefix external or global name spaces and flag these up to each other when missed.